### PR TITLE
write to DEPLOYED_AT file after rspec runs

### PR DESCRIPTION
**Pull request by JzGo**
- Relates to ticket #143 Providing Deployment Timestamp accessible to apps running in the cluster
- Writes a timestamp into `DEPLOYED_AT` after rspec runs

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -40,6 +40,9 @@ jobs:
           # Excludes tests that require Yale VPN access
           command:  |
             bundle exec rspec --tag ~vpn_only:true
+      - run:
+          name: timestamp
+          command: echo $(date -u +%FT%TZ) >> DEPLOYED_AT
```
Comment (mlooney) [inline .circleci/config.yml:45]: is this file accessible in the web app somehow?  could we use a similar process to BLACKLIGHT_VERSION in the ecs-compose (where env var is injected at deploy time, then referenced in template)

Comment (mark-dce) [reply .circleci/config.yml:45]: I'd like a timestamp anytime a container is spun up, so I think the main idea here of exporting an environment variable is great.  I'm wondering if setting it in the docker-compose.yml makes more sense - that would give us consistent behavior across development and deployment environments.

Comment (JzGo) [reply .circleci/config.yml:45]: It should be. It's intended it to be read on initialization (some work was done for that here: config/initializers/git_sha.rb) Might need some tweaking as I can't test locally.

If this works as is, I would think it's possible for app versions to be set up in ecs-compose and read out similarly on app initialization.

**Comment by fnibbit** [comment]
I intend to to try this out on an ecs cluster in the morning, suddenly realized I was in work-all-evening mode.  :)

**Comment by JzGo** [comment]
Updated PR: Moved timestamp write to compose stage which I think resembles more closely what we could do at the ecs-compose level.

**Pull request merged**